README.md

```diff
@@ -285,4 +285,4 @@ This project is [MIT](./LICENSE) licensed.
 
 _NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
 
```

Comment:
- [ ] Kindly customize your `README.md` file according to your project details, features, tech stacks, future-feature and every other important thing. Please always consider the `README.md` file just like a manual of some instructions or information regarding the project, so that a user can be able to understand your project properly and work as a collaborator or experiment with it. I would really like to suggest you few things which are as follows:

  - Except for the *FAQ* section rest of everything is important and related to the project.
  - Please again don't forget that you must have to customize each and every detail of your `README.md` file.
  - Because I am not able to make inline-comment on your `README.md` file to tell everything more precisely I would like to share [this](https://github.com/skyv26/portfolio) link which will help you to visualize and understand how you can improve your `README.md` more professionally.
  - Please it is important don't put any broken link or invalid path in your `README.md` file. License is very important to add in your every repo and I saw that your license link is not working because you haven't added valid path.